src/nemo_evaluator/reports/trajectories.py

```
@@ -107,6 +134,206 @@ def _read_jsonl(path: Path) -> list[dict[str, Any]]:
     return rows
 
 
+def _truncate_text(value: Any, limit: int = _EXAMPLE_TEXT_CHARS) -> str:
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)